### DISCUSSION TITLE
Convert 3.2.3-beta (2011) to current

### DISCUSSION THREAD

**Discussion by mca222** [wxFormBuilder/wxFormBuilder#849] [Q&A]
We've have an ancient (2011) template created in 3.2.3-beta. I tried working in this version, but it has the tendency to crash very often. As such I want to work in the latest and greatest 4.2.1 version.

But if I open the old .fbp-file it gives the warning below. But since I get the warning 6500 times, I cannot check all.

So is there a list of warning I can safely ignore? Or can wxFormBuilder create a .fbp-file from the generated c++ source? Or any other ways forward?
```
The property "foo" of class "bar" is not supported by this version of wxFormBuilder.
If your project file was just converted from an older version, the conversion is not complete.
Otherwise, this project file is from a newer version of wxFormBuilder.

The value of the property is: "baz"

If you save this project, YOU WILL LOSE DATA!
```

**Comment by sodevel** [comment]
This warning happens if the fbp-File contains elements that are not part of the current wxFB data model. The project file conversion code should update old project files to prevent such messages. Since you are using a very old wxFB version your fbp-File uses a very old version too, there are multiple possibilities why these messages occur:

1. These properties were simply removed and no one wrote conversion code for this case
2. The conversion code was always broken for your project file version
3. I broke the conversion code when replacing the XML library for wxFB 4.0.0
4. Your project file version is from the pre-UTF-8 age

Since you get so many of such messages, these properties might be of a common base clase like `wxWindow`, case 1. might apply, but without knowing what "foo" and "bar" are, this is hard to say. If case 4. applies to you, you could use the latest wxFB 3.x version, this might also help in case 3.

If you can share your fbp-File, the easiest for you would be to upload the file here so i can take a look. If thats not possible, then at least i need to know which file version your fbp-File has and what at least some "foo" and "bar" are. A fbp-File is just a XML-File you can open in a text editor, i need the information from its top, it should look like this (depending on the file version the version properties might be further down and structured differently):

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<wxFormBuilder_Project>
  <FileVersion major="1" minor="18"/>
```
